packages/search-ui-workplace-search-connector/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@elastic/search-ui-workplace-search-connector",
+  "version": "1.10.0-rc.1",
+  "description": "A Search UI connector for Elastic's Workplace Search Service",
+  "license": "Apache-2.0",
+  "main": "lib/cjs",
+  "module": "lib/esm",
+  "types": "./lib/esm/index.d.ts",
+  "sideEffects": false,
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "files": [
+    "lib"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/elastic/search-ui.git",
+    "directory": "packages/search-ui-workplace-search-connector"
+  },
+  "scripts": {
+    "test-ci": "jest --runInBand",
+    "test": "jest",
+    "clean": "rimraf es lib",
+    "watch-js": "./bin/watch-js",
+    "build-js": "./bin/build-js",
+    "build": "run-s clean build-js",
+    "watch": "run-p watch-js",
+    "prepare": "npm run build"
+  },
+  "bugs": {
+    "url": "https://github.com/elastic/search-ui/issues"
+  },
+  "devDependencies": {
+    "npm-run-all": "^4.1.5",
+    "rimraf": "^2.6.3",
+    "typescript": "^4.5.4"
+  },
+  "dependencies": {
+    "@elastic/app-search-javascript": "^7.13.1",
```

Comment:
Should be able to remove dep

Reply:
Nice catch, removed https://github.com/elastic/search-ui/pull/650/commits/12a480cfd65e2563d9f8972176d785b7869bc887